setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="confluent-kafka-helpers",
-    version="0.10.5",
+    version="1.0.0.dev0",
```

Comment:
I'll evaluate this dev release in a few services for a week or so. When I've verified that everything works as expected I will create a stable release 1.0.0